---
layout: page
title: Program for BioVis@Vis Workshop
permalink: /program_ieee/
back_title: IEEE VIS
back_url: ieeevis
---

## BioVis@Vis 2019 Challenges Workshop Program

**When:** Thursday October 24nd, 2019, 9:00 - 12:20

**Location:** Room 1 (for further details see the [http://ieeevis.org/](http://ieeevis.org/))

<a name="agenda"></a>

<hr>
<div>
    <div class="sumTime2"> 9:00 - 9:10</div>
    <div>
        <div class="sumContent">Opening remarks and BioVis @ISMB Recap</div>
    </div>
    <!-- <div class="sumDetail" style="padding-left:120px;"><i>Session Chair: Ana & Cyndey</i> </div> -->
    <!-- <div class="sumDetail" style="padding-left:120px;font-size:12px;"><i>(* indicates presenting author)</i> </div> -->
</div>

<hr>
<div>
    <div class="sumTime2"> 9:10 - 10:30</div>
    <div>
        <div class="sumContent">Domain Expert Presentations</div>
    </div>
    <!-- <div class="sumDetail" style="padding-left:120px;"><i>Session Chair: Ana & Cyndey</i> </div> -->
    <!-- <div class="sumDetail" style="padding-left:120px;font-size:12px;"><i>(* indicates presenting author)</i> </div> -->
</div>


<div>
    <div class="sumTime2" style="font-size:15px;color:#21a186;">9:10 - 9:35</div>
    <div>
        <div class="sumContent" style="padding-left:120px;font-size:15px;color:#21a186;">Visual Innovations and Challenges in Drug Development</div>
          <div class="sumDetail" style="padding-left:120px;font-size:15px;color:#21a186;"><i> Speaker: <b> Martin Karpefors </b>, AstraZeneca</i> <a href = "#d_martin_abs" style ="font-size:10px;color:#21a186;"> ... [abstract & bio]</a></div> 
    </div>
</div>


<div>
    <div class="sumTime2" style="font-size:15px;color:#21a186;">9:35 - 10:05</div>
    <div>
        <div class="sumContent" style="padding-left:120px;font-size:15px;color:#21a186;">Personalized OncoGenomics: Translating Whole Cancer Genomes for the Clinic</div>
          <div class="sumDetail" style="padding-left:120px;font-size:15px;color:#21a186;"><i> Speaker: <b> Erin Pleasance</b>, British Columbia Cancer Agency</i> <a href = "#d_erin_abs" style ="font-size:10px;color:#21a186;"> ... [abstract & bio]</a></div> 
    </div>
</div>


<div>
    <div class="sumTime2" style="font-size:15px;color:#21a186;">10:05 - 10:35</div>
    <div>
        <div class="sumContent" style="padding-left:120px;font-size:15px;color:#21a186;"> Opportunities and Challenges for Visualizing Human Tumor Atlases</div>
          <div class="sumDetail" style="padding-left:120px;font-size:15px;color:#21a186;"><i> Speaker: <b> Sean E. Hanlon</b>, National Cancer Institute</i> <a href = "#d_sean_abs" style ="font-size:10px;color:#21a186;"> ... [abstract & bio]</a></div> 
    </div>
</div>

<hr>

<!-- COFEE BREAK -->
<div>
  <div class="sumTime" style="padding-top:5px;"> 10:35 - 10:50 </div>
    <div class="ttile" style="padding-left:120px; padding-top:5px;"> <em>Coffee Break </em></div>
</div>  
<hr>



<!-- DATA CHALLENGE SESSIONS -->
<div>
    <div class="sumTime2"> 10:50 - 11:40</div>
    <div>
        <div class="sumContent">Data and Redesign Challenge Results</div>
    </div>
</div>

<div>
  <div class="sumTime" style="padding-top:5px;"> 10:50 - 11:00 </div>
    <div class="ttile" style="padding-left:120px; padding-top:5px;"> <em> Data and Redesign Challenge Overview </em></div>
</div>  

<!-- DATA CONTEST : JAMES -->
 <div>
      <div class="sumTime" style="padding-top:5px;"> 11:00 - 11:07</div>
      <div>
          <div class="ttile" style="padding-left:120px; padding-top:5px;"> A Radial View of the BioVis Challenge 2019 Dataset <b></b></div>
      </div>
      <div class="sumDetail" style="padding-left:120px;"> <em>Presenter: James Scott-Brown</em> </div>
  </div>

  <!-- DATA CONTEST : HackSeq -->
 <div>
      <div class="sumTime" style="padding-top:5px;"> 11:08 - 11:20</div>
      <div>
          <div class="ttile" style="padding-left:120px; padding-top:5px;"> HackSeq 2019 Results <b></b></div>
      </div>
      <div class="sumDetail" style="padding-left:120px;"> <em>Presenter: Ana Crisan</em> </div>
  </div>


  <!-- REDESIGN CONTEST : Cassia -->
 <div>
      <div class="sumTime" style="padding-top:5px;"> 11:20 - 11:30</div>
      <div>
          <div class="ttile" style="padding-left:120px; padding-top:5px;"> RasVis: Redesigning Pathways to Support Analyses of Experimental Data <b></b></div>
      </div>
      <div class="sumDetail" style="padding-left:120px;"> <em>Presenter: Cassia Artanegara and Angus Forbes</em> </div>
  </div>

  <!-- REDESIGN CONTEST : Martin -->
 <div>
      <div class="sumTime" style="padding-top:5px;"> 11:30 - 11:40</div>
      <div>
          <div class="ttile" style="padding-left:120px; padding-top:5px;"> The Tendril Plot – A Redesign supporting more effective safety assessment following a clinical trial <b></b></div>
      </div>
      <div class="sumDetail" style="padding-left:120px;"> <em>Presenter: Martin Karpefors </em> </div>
  </div>

  <hr>

  <!-- CAPSTONE  -->
  <div>
    <div class="sumTime2" style="font-size:15px;color:#21a186;">11:40 - 12:15</div>
    <div>
        <div class="sumContent" style="font-size:15px;color:#21a186;">Captsone</div>
        <div class="sumContent" style="padding-left:120px;font-size:15px;color:#21a186;">Little table, what do you have to say of visualization?</div>
          <div class="sumDetail" style="padding-left:120px;font-size:15px;color:#21a186;"><i> Speaker: <b> Martin Krzywinski</b>, Michael Smith Genome Sciences Centre</i><a href = "#c_martin_abs" style ="font-size:10px;color:#21a186;"> ... [abstract & bio]</a></div>
    </div>
</div>
<hr>
<div>
    <div class="sumTime2"> 12:15 - 12:20</div>
    <div>
        <div class="sumContent">Closing Remarks</div>
    </div>
    <!-- <div class="sumDetail" style="padding-left:120px;"><i>Session Chair: Ana & Cyndey</i> </div> -->
    <!-- <div class="sumDetail" style="padding-left:120px;font-size:12px;"><i>(* indicates presenting author)</i> </div> -->
</div>
<hr>




### Domain Expert Presentations

Expert panelists will discuss the data analysis and visualization challenges in their specific areas. Each panelist will deliver a short presentation, after which audience members will have an opportunity to further discuss with them about their specific challenges and existing solutions.

<!-- Martin Karpefors --->
<div class="talk">
    <div class="ttitle">Visual Innovations and Challenges in Drug Development</div> 
    <div><span class="tspeaker"><a href="_">Martin Karpefors</a></span>, <span>AstraZeneca</span></div>

   <div class="tportrait"><img src="{{ site.baseurl}}/images/speakers/MartinKarpefors.jpg" style="width: 150px;" alt="Martin Karpefors"></div>

    <div class="tbioabstract" id="d_martin_abs">

      <div class="tabstract"><b>Abstract:</b>
        The drug development process requires a drug candidate to be repeatedly tested, compared and evaluated. To make the right evidence-based decisions of the future potential of the drug, balancing both efficacy and safety, it is important to take all relevant data into account. Mathematical models in combination with graphical representations and interactive visualizations are key to create an overview and understanding of the huge amount of complex data. In this talk I will provide examples of recent progress and remaining visualization challenges.

        One example of recent advancement is the Tendril plot – a new type of visualization that overcomes some of the current limitations of adverse event analysis in clinical trials. The interactive visualization offers an innovative and effective way for clinical teams to graphically explore, interpret and present adverse event data. By providing clear visual guidance that helps prioritize patient safety findings, this work addresses an important business need and saves both time and resources. 

        A visual challenge that remains to be solved is to clearly visualize the different aspect of patient journeys through clinical trials, which are comprehensible to the patients and meaningful to the researchers. It will require new digital designs to manage the visual complexity both on an aggregated and individual level, e.g. representing each patient’s own sequence of medication, adverse events, laboratory test results, medical sensor input, etc. In an increasingly connected world, clinical trials will collect even more data, raising this visual analytics challenges even more.
        </div>
    </div>

    <div class="tbioabstract">

    <div class="tabstract" id="d_martin_bio"><b>Speaker Bio:</b>
       Dr. Martin Karpefors works as Senior Biomedical Informatics Director in the Data Science and AI department at AstraZeneca R&D Gothenburg, Sweden. 

        Dr. Karpefors graduated with a M.Sc. in Physics from Chalmers University of Technology in 1995. In 2000 he finished his PhD studies in biochemistry and biophysics, defending his thesis entitled "Control Mechanisms of Electron and Proton Transfer in Cytochrome c Oxidase". Later that year, Dr. Karpefors joined AstraZeneca. Since then, he has been active within the fields of mathematical modeling, machine learning, visualization and informatics to support decision making in early and late clinical development, especially within the cardiovascular, metabolic and respiratory disease areas.
        
    </div>

    </div>
    
</div>

<!-- Sean Hanolon --->
<div class="talk">
    <div class="ttitle">Opportunities and challenges for visualizing Human Tumor Atlases</div>
    <div><span class="tspeaker"><a href="https://cssi.cancer.gov/about-us/leadership/sean-e-hanlon-phd">Sean E. Hanlon</a></span>, <span>National Cancer Institute</span></div>

   <div class="tportrait"><img src="{{ site.baseurl}}/images/speakers/Hanlon_2018.jpg" style="width: 150px;" alt="Sean Hanlon"></div>

    <div class="tbioabstract">

      <div class="tabstract" id="d_sean_abs"><b>Abstract:</b>
        Emerging single-cell and in situ technologies are facilitating the characterization of normal and diseased human cells and tissues at unprecedented resolution. Coupling these genomics data with imaging modalities that provide information about tissue composition, gross organ structure, and metabolism can improve our understanding of the development and evolution of cells and tissues. Several recent initiatives have focused on generating ‘atlases’ that integrate multi-scale maps to facilitate our understanding of health and disease. These include the 4-Dimensional Nucleome (4DN) Program, the Human BioMolecular Atlas Program (HuBMAP), the Human Cell Atlas (HCA) initiative, and the Human Tumor Atlas Network (HTAN). These atlas building programs seek to integrate multi-scale data from 3D genome organization to single cell epigenetic states and expression profiles to spatial transcriptomics and protein localization to cell state and immune status to tissue level organization and phenotypes. The ultimate aim is to integrate multi-modal and multi-scale data and provide it in a user-friendly environment for the scientific research and clinical communities and the general public. A major challenge to developing comprehensive cell and tissue atlases is the visualization of data across scales and modalities. I will share highlights of the HTAN program and data and discuss opportunities and challenges for visualizing HTAN data and atlases.
        </div>
    </div>

    <div class="tbioabstract">

    <div class="tabstract" id="d_sean_bio"><b>Speaker Bio:</b>
        Dr. Sean E. Hanlon is an Associate Director of the Center for Strategic Scientific Initiatives (CSSI) at the NCI where he contributes to the vision and strategic plans of the Center, provides leadership in the analysis and evaluation of emerging fields, and develops and implements new initiatives. Additionally, Dr. Hanlon serves as a CSSI/NCI representative on NCI, NIH, and inter-agency working groups and committees, including the trans-NCI Data Sharing working group. He also facilitates collaborations and provides strategic and scientific leadership to collaborative transdisciplinary programs, including the NIH Common Fund’s 4D Nucleome program.

        Prior to joining CSSI, Dr. Hanlon served as Program Director within the NCI Division of Cancer Biology where he served as Director of the Physical Sciences-Oncology Network (PS-ON). In this role, he led the scientific management and oversight of the PS-ON and worked to identify synergistic opportunities and foster new collaborations. He also managed the PS-ON Trans-Network Projects program and the PS-ON Data Coordinating Center which together leveraged the expertise of multiple teams to test new physical sciences-based cancer questions and fostered the sharing of PS-ON generated data.
        
    </div>

    </div>

    
</div>


<!-- Erin Pleasance--->
<div class="talk">
    <div class="ttitle">Personalized OncoGenomics: Translating Whole Cancer Genomes for the Clinic</div>
    <div><span class="tspeaker"><a href="http://www.bcgsc.ca/faculty/sjones/research-team-members/Erin01.JPG/view">Erin Pleasance</a></span>, <span>British Columbia Cancer Agency</span></div>

   <div class="tportrait"><img src="{{ site.baseurl}}/images/speakers/ErinPleasancePhoto.jpg" style="width: 150px;" alt="Erin Pleasance"></div>

    <div class="tbioabstract">

      <div class="tabstract" id = "d_erin_abs"><b>Abstract:</b>
        Personalized cancer medicine aims to tailor treatment to an individual's tumour genetics, guiding and providing new avenues for therapy. At BC Cancer,the Personalized Oncogenomics (POG) program generates whole genome and transcriptome data from patients with advanced cancers, and integrates this with domain knowledge to propose therapeutic targets. These targets are discussed with treating oncologists in a molecular tumour board and can inform treatment decisions. The breadth of data generated poses visualization challenges; for each patient there are available mutations, expression, and clinical data, in addition to comparisons between patient data and existing databases. Visualizations currently in use include plots focusing on specific analyses, pathway-style visual summaries, and a standardized report format. The target users of the data are also broad, encompassing genomics researchers, pathologists, and clinicians focused on patient care, adding to challenges in data reporting. Effectively communicating detailed analyses while maintaining simplicity in interpretation is essential for successfully translating genomic data into clinical impact.
      

        </div>
    </div>

    <div class="tbioabstract">

    <div class="tabstract"><b>Speaker Bio:</b>

        Dr. Erin Pleasance is a Staff Scientist with the BC Cancer Agency and Micheal Smith Genome Sciences Centre. Her research is focused on the use of genomics for cancer medicine. Dr. Pleasance is currently involved in the Personalized OncoGenomics (POG) project, a collaboration between researchers in bioinformatics and genomics, clinicians, and pathologists that uses genomic technologies to contribute to personalized cancer therapy decision making at the BC Cancer Agency. In recent years, she have also been involved in The Cancer Genome Atlas (TCGA) and various cancer sequencing projects funded through the National Cancer Institute. She completed her PhD in medical genetics at the GSC with Dr. Steve Jones, followed by postdoctoral work at the Sanger Institute in the UK with Sir Mike Stratton, sequencing some of the first cancer genomes. 
    </div>

    </div>
</div>


### Capstone 

<!-- Martin Kyrwinsky --->
<div class="talk">
    <div class="ttitle">Little table, what do you have to say of visualization? </div> 
    <div><span class="tspeaker"><a href="http://mkweb.bcgsc.ca/">Martin Krzywinski</a></span>, <span>Michael Smith Genome Sciences Centre</span></div>

   <div class="tportrait"><img src="{{ site.baseurl}}/images/speakers/martin_krzywinski-2.jpg" style="width: 150px;" alt="Martin Krzywinski"></div>

    <div class="tbioabstract" id = "c_martin_abs">

      <div class="tabstract"><b>Abstract:</b>

      <blockquote><em>
      Little P value <br>
	    What do you have to say<br>
	    Of significance?<br>
	      — Steve Ziliak<br>
      </em></blockquote>

      The job of the table is to classify and organize data. This is achieved by alignment, spacing and typographical hierarchy. Visualizations extend tabularization by adding elements to make quantitative proportions and trends more salient. At the heart of an effective visualization is an effective (if subtle) table. <br>

      Thus, any inquiry into visualization practices must include questions about tabularization practices. If we want good visualizations, we want good tables. Do we make good tables? Generally, we do not. <br><br>

      Good tables don’t elicit the same respect and satisfaction as good plots. It’s hard to rally support for rows and columns of text and no color. The problem is that if you love cake but find flour boring, you will never understand cake. <br><br>

      Using examples from classrooms, conferences and the literature, I’ll demonstrate the benefit of seeing visualizations are tables with colors and shapes and how this view can shape the process of creation, diagnosis, and redesign. <br>

        </div>
    </div>

    <div class="tbioabstract">

      <div class="tabstract"><b>Speaker Bio:</b>

        Martin Krzywinski is a staff scientist at the BC Cancer Agency  & Michael Smith Genome Sciences Centre in Vancouver, working at the intersection of science and design. Martin is perhaps best known for creating popular visualization techniques including Circos and Hive Plots. For more, see <a href= "http://mkweb.bcgsc.ca/bio/martin-krzywinski-bio.pdf"> Martin’s official biography </a>.

        </div>
    </div>
</div>
